### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by bocajthomas**
Updated to V2.0.0 feature list 
Will change as new versions (Ci builds) of v2 come out. Ready for stable release

### CODE REVIEW

**Comment by authorisation** [comment]
Hey
About the Installation Guides, feel free to send them in here as a file as Github does not support PRs to the wiki
Thank you

**Comment by bocajthomas** [comment]
Installation Guide V2 In Detail 
[InstallationGuide.md](https://github.com/rhunk/SnapEnhance/files/13757399/InstallationGuide.md)

**Comment by bocajthomas** [comment]
Installation Guide V2 In Detail 
Make Changes If You Must ✨️ 
[InstallationGuide.md](https://github.com/rhunk/SnapEnhance/files/13757418/InstallationGuide.md)

**Comment by authorisation** [comment]
I put the installation guide on the wiki, I'll still change up some stuff later though
Thanks !!

**Comment by bocajthomas** [comment]
> I put the installation guide on the wiki, I'll still change up some stuff later though Thanks !!

I'll be adding to it when it's needed

**Comment by authorisation** [comment]
I'll be merging soon, add yourself to the contributors list if you wish.

**Comment by bocajthomas** [comment]
> I'll be merging soon, add yourself to the contributors list if you wish.

Thank you 😊 
I have added my self to the " contribution " list

File: README.md [hunk 1]
```diff
@@ -180,7 +181,7 @@ We no longer offer official LSPatch binaries for obvious reasons. However, you'r
 <details>
   <summary>My phone isn't rooted; how do I use this?</summary>
   
-  - You can use LSPatch in combination with SnapEnhance to run this on an unrooted device, however this is unrecommended and not considered safe.
+  - Check The Wiki Page For Guides
```
Comment (authorisation) [inline README.md:187]: revert this!

**Comment by authorisation** [comment]
:+1: approved

File: README.md
Comment (bocajthomas) [inline README.md:1]: How Come ?

**Pull request closed without merging**